R/sample_predictions.R

```diff
@@ -300,3 +345,31 @@ sample_nowcasts <- function(point_nowcast_matrix,
   draws_df <- Reduce(rbind, draws_df_list)
   return(draws_df)
 }
+#' Sample from negative binomial model given a set of predictions
+#'
+#' @param pred Vector of predictions. Default is `NULL`
+#' @param uncertainty_params Vector of uncertainty parameters. Default is
+#'     `NULL`.
```

Comment:
why null is this pattern documented somewhere i.e where it is needed as an input